pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -350,6 +350,14 @@ def test_uint64_type_handling(dtype, how):
     tm.assert_frame_equal(result, expected, check_exact=True)
 
 
+def test_func_duplicates_raises():
```

Comment:
Is this duplicating an existing test?

Reply:
emm, you mean the name of the test? or the test itself? I dont think it's duplicating an existing test, at least i look up the tests and corresponding error message, this doesn't seem to be a duplicate to me. @TomAugspurger